src/docs/arch-storage-extension.dox

```diff
@@ -0,0 +1,104 @@
+/*! @arch_page arch-cloud-storage-extension Storage Source
+
+@section extension_storage_source Cloud Storage Source Implementation
+
+The WiredTiger extension allows the control of how files are stored via the storage source
+abstraction layer. More information can be found through @ref custom_storage_sources. There are
+three storage sources that extends the WiredTiger's capability to store the data files onto the
+cloud storage through Azure, Google and AWS providers. The cloud storage extensions provides a
+means to construct, configure and control a custom file system linked to a cloud bucket. All three
+storage source implementations internally interacts with the cloud storage through their own C++
+SDKs.
+
+All storage sources implements the following key functionalities defined by the storage source 
+interface:
+- Initialize the store, the AWS SDK and create a logging module for the store and the SDK.
```

Comment:
This point should no longer mention `AWS`.

Reply:
thanks! removed